di_website/templates/publications/chapter-nav/nav-item.html

```diff
@@ -18,8 +18,20 @@
             data-current-chapter="1"
         {% endif %}
     >
-      <span class="chapter-title">{{ item.label | title }}:</span>
-      <span class="chapter-nav__title">{{ item.title }}</span>
+      <span class="chapter-title">
+          {% if item.is_foreword %}
+              {{ item.label_type | capfirst }}
+          {% else %}
+              {{ item.label }}
+          {% endif %}
+      </span>
+      <span class="chapter-nav__title">
+          {% if item.is_summary %}
+              {{ item.label | capfirst }}
+          {% else %}
+              {{ item.title }}
+          {% endif %}
+      </span>
```

Comment:
I don't see the reason for this change either ... This is meant for the title ... no one's said that Summary pages should only have titles `Executive summary` ... the label tells you it's a summary ... title can be anything.